apps/src/code-studio/pd/application/principalApproval/PrincipalApprovalComponent.jsx

```diff
@@ -348,14 +301,31 @@ const PrincipalApprovalComponent = props => {
             learning program and teaching a new course. That’s why your approval
             is required for the teacher's application to be considered.
           </p>
+          <p>
+            Please note that we are not able to consider the teacher for
+            acceptance into the Professional Learning Program until you have
+            submitted this approval form.
+          </p>
           <LabeledSelect
             name="title"
             required={false}
             placeholder="Select a title"
           />
           <LabeledInput name="firstName" />
           <LabeledInput name="lastName" />
+          <LabeledInput name="role" />
           <LabeledInput name="email" />
+          <LabeledRadioButtons
+            name="canEmailYou"
+            label={
+              <span>
+                {PageLabels.canEmailYou}{' '}
+                <a href={'https://code.org/privacy'}>
```

Comment:
Nit: It looks like from the video you provided, this opens in the same window. Could you use our Design System's `Link` component with the `openInNewTab` prop https://github.com/code-dot-org/dsco_/blob/bff9c303f7bc7ad7b0541f732267597342b09495/packages/link/lib/Link.jsx#L16?

Reply:
Good call, will do!

Reply:
Done with a new video showing that it opens in a new tab!